src/gui/dialogs/multiplayer/mp_create_game.cpp

```diff
@@ -472,10 +513,15 @@ void mp_create_game::on_tab_select(window& window)
 	}
 }
 
-void mp_create_game::on_mod_toggle(const int index)
+void mp_create_game::on_mod_toggle(window& window, const int index, toggle_button* sender)
 {
-	create_engine_.set_current_mod_index(index);
-	create_engine_.toggle_current_mod();
+	(void)sender;
```

Comment:
Why is this here?

Reply:
Looks like an unused parameter. So the question is why was the parameter needed. I'd assume to fix a required function signature.